custom_dc/load_data.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+# Script to process stats using simple stats loader.
+# Defaults
+MODE="customdc"
+OUTPUT_DIR=".data/output_$MODE"
+USAGE="Script to load data into custom dc using simple importer.
```

Comment:
What is the purpose of this variable?

Reply:
OK, please rewrite "custom dc" as "custom Data Commons"

Reply:
Done